apps/console/src/app.tsx

```diff
@@ -70,9 +70,12 @@ export const App: FunctionComponent<{}> = (): ReactElement => {
     const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
     const allowedScopes: string = useSelector((state: AppState) => state?.auth?.scope);
     const appTitle: string = useSelector((state: AppState) => state?.config?.ui?.appTitle);
+    const UUID: string = useSelector((state: AppState) => state.profile.profileInfo.id);
```

Comment:
Lets convert this variable name to lowercase (camel case) as this is not a hard coded final constant.

Reply:
Fixed in #2729